frontend/src/component/DetailsProduct.tsx

```diff
@@ -19,6 +22,16 @@ export default function DetailsProduct(props: DetailsProductProps) {
         }
     }, [id, props])
 
+    const handleSave = () => {
+        if (thisProduct && props.actualOrderDetailsItems) {
+            const orderItem: OrderDetailsItem = {product: thisProduct, count: 1, price: thisProduct.price}
```

Comment:
wird count danach nicht hardkodiert? is es in Plan? :-)

Reply:
jau hehe ;)